src/commands/jetpack-module.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Team51\Command;
+
+use Team51\Helper\API_Helper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\ProgressBar;
+
+
+class Jetpack_Module_Search extends Command {
+	protected static $defaultName = 'module-search';
```

Comment:
I'm scratching my head on this one... how did I even test that? 
I think I made the change at the last minute, maybe because it's too close to `jetpack-modules` in #78 
The other option is to rename both, so like `jetpack-module-list` and `jetpack-module-search` - longer, but more explicit. What do you think?

Reply:
I think `jetpack-module-list` and `jetpack-module-search` are ok.

Since the second one returns a list of sites, it could also be something like `jetpack-sites-with adwords on`.

In the end, users we'll get used to whatever name is chosen. We could also discuss creating a new issue to revisit the naming of all existing commands, if the team agrees (for instance, I really want to change `remove-user` to `offboard-collaborator`)